include/gridtools/meta/type_traits.hpp

```diff
@@ -41,52 +41,5 @@ namespace gridtools {
     struct disjunction<T, Ts...> : std::conditional<bool(T::value), T, disjunction<Ts...>>::type {};
 
     template <typename... Ts>
-    struct void_t_impl {
-        using type = void;
-    };
-    template <typename... Ts>
-    using void_t = typename void_t_impl<Ts...>::type;
```

Comment:
`void_t` is C++17 feature. Please restore

Reply:
Yes. `void_t` is still there, I just removed `void_t_impl` because it is not needed.

Reply:
Quote from https://en.cppreference.com/w/cpp/types/void_t:
```
Until CWG 1558 (a C++14 defect), unused parameters in alias templates were not guaranteed to ensure SFINAE and could be ignored, so earlier compilers require a more complex definition of void_t, such as
...
```
I am OK of having simplified definition of `void_t`, but we need to add the test in this case.

Reply:
Ah I see - didn't read that... added a test.